src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -81,6 +81,15 @@ public void Configure(TelemetryConfiguration configuration)
             {
                 module.Initialize(configuration);
             }
+
+            foreach (ITelemetryProcessor processor in configuration.TelemetryProcessors)
+            {
+                ITelemetryModule module = processor as ITelemetryModule;
```

Comment:
It feels wrong to use ITelemetryModule to get an Initialize method if it isn't really a telemetry module.